staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -867,6 +867,10 @@ const (
 
 	// VirtualMachineNameLabel is the name of the Virtual Machine
 	VirtualMachineNameLabel string = "vm.kubevirt.io/name"
+
+	// PVCMemoryDumpAnnotation is the name of the memory dump representing the vm name,
+	// pvc name and the timestamp the memory dump was collected
+	PVCMemoryDumpAnnotation string = "kubevirt.io/memory-dump"
```

Comment:
Just a question, what happens to an old dump? Is this deleted or do we lose its reference but it can be listed as PVC?

Reply:
not sure I understood the question. If we do another memory dump to the same PVC the annotation value will be changed to the new memory dump file name (the older memory dump is deleted if doing another memory dump to the same PVC)